vmaas/load.go

```diff
@@ -684,18 +684,28 @@ func loadString(c *Cache) {
 func loadDBChanges(c *Cache) {
 	defer utils.TimeTrack(time.Now(), "DBChange")
 
-	rows := getAllRows("dbchange", "*")
-	arr := []DBChange{}
-	var item DBChange
-	for rows.Next() {
-		err := rows.Scan(&item.ErrataChanges, &item.CveChanges, &item.RepoChanges,
-			&item.LastChange, &item.Exported)
+	var timestamps [5]string
+	row := sqlDB.QueryRow("SELECT * FROM dbchange LIMIT 1")
```

Comment:
I know it wasn't there before but it would be better to list column names instead of `SELECT *` to be sure we're assigning correct timestamp to correct struct attribute